src/necessities.md

```diff
@@ -109,3 +109,42 @@ cannot be used in some situations where most of the Rust runtime stack can.
 The license of a crate's dependencies can affect the restrictions on
 distribution of the crate itself, so a permissively-licensed crate should
 generally only depend on permissively-licensed crates.
+
+<a id="c-msrv"></a>
+## Crate has an MSRV policy (C-MSRV)
+
+A crate should clearly document its Minimal Supported Rust Version:
+
+* Which versions versions of Rust are supported now?
+* Under what conditions is MSRV increased?
+* How are MSRV increases reflected in the semver version of the crate?
+
+Compliance with a crate’s stated MSRV should be tested in CI.
+
+The API guidelines tentatively suggest that, for libraries, an MSRV increase should
+*not* be considered a semver-breaking change. Specifically, when increasing
+MSRV:
+
+* for crates past `1.0.0`, increment minor version (`1.1.3` -> `1.2.0`),
+* for crates before `1.0.0`, increment patch version (`0.1.3` -> `0.1.4`).
+
+This reduces the amount of ecosystem-wide work for MSRV upgrades and prevents
+incompatibilities. It also is a de-facto practice for many cornerstone crates.
+This policy gives more power to library consumers to manually select working
+combinations of library and compiler versions, at the cost of breaking `cargo
+update` workflow for older compilers.
+
+However, do not increase MSRV without a good reason, and, if possible, batch
+MSRV increases with semver-breaking changes.
+
+Nonetheless, some crates intentionally choose to treat MSRV increases as a semver
+breaking change. This is also a valid strategy, but it is not recommended as the
+default choice.
+
+To reliably test MSRV on CI, use a dedicated `Cargo.lock` file with dependencies
+pinned to minimal versions:
```

Comment:
Some crate bumps MSRV of the crate itself due to the dev-dependency MSRV bump, but in some cases, the crate itself can be compiled with an older version, even if the dev-dependency requires a newer version.
I'm not sure if it's really preferable to recommend `cargo test` instead of `cargo build` here.
AFAIK there are many projects that do not run tests on MSRV. (e.g., [serde](https://github.com/serde-rs/serde/blob/e3d871ff7bf10dadf10bdc234a55692228358d0e/.github/workflows/ci.yml#L63-L78), [syn](https://github.com/dtolnay/syn/blob/2c83686bede4f64353e9ad10033a23f77e56eb94/.github/workflows/ci.yml), [regex](https://github.com/rust-lang/regex/blob/3bcae3b6954ea3628ef0974f0a2604a6ae248408/.github/workflows/ci.yml#L99-L103), [tokio](https://github.com/tokio-rs/tokio/blob/479c545c20b2cb44a8f09600733adc8c8dcb5aa0/.github/workflows/ci.yml#L182), [futures](https://github.com/rust-lang/futures-rs/blob/master/.github/workflows/ci.yml), etc...)

Reply:
Yeah, good suggestion, applied it!